alloc-fmt/src/bin/assert.rs

```diff
@@ -0,0 +1,36 @@
+// Copyright 2017 the authors. See the 'Copyright and license' section of the
+// README.md file at the top-level directory of this repository.
+//
+// Licensed under the Apache License, Version 2.0 (the LICENSE-APACHE file) or
+// the MIT license (the LICENSE-MIT file) at your option. This file may not be
+// copied, modified, or distributed except according to those terms.
+
+#[macro_use]
+extern crate alloc_fmt;
+
+#[cfg_attr(feature = "cargo-clippy", allow(cyclomatic_complexity))]
+fn main() {
```

Comment:
For these binaries, do you think it's worth setting #[global_allocator] to something that panics in alloc, or is that overkill

Reply:
I think that's overkill; I've tested that functionality by merging this with the current `tls` branch and seeing what happens since that branch has existing assertion failures.